data-manipulation/encryption/rc4/encrypt-data-using-rc4-with-custom-key-via-winapi.yml

```
@@ -0,0 +1,29 @@
+rule:
+  meta:
+    name: encrypt data using RC4 with custom key via WinAPI
+    namespace: data-manipulation/encryption/rc4
+    author: blaine.stancill@mandiant.com
+    scope: function
+    att&ck:
+      - Defense Evasion::Obfuscated Files or Information [T1027]
+    mbc:
+      - Defense Evasion::Obfuscated Files or Information::Encryption-Standard Algorithm [E1027.m05]
+      - Cryptography::Encrypt Data::RC4 [C0027.009]
+    references: https://www.phdcc.com/cryptorc4.htm
```

i think this should be a list
```suggestion
    references: 
      - https://www.phdcc.com/cryptorc4.htm
```